### PR TITLE
docs: Add WoeUSB as an alternative tool to burn Windows USB images

### PR DESCRIPTION
**Pull request by Lin-Buo-Ren**
By far it is one of the few applications that support Windows image and runs on GNU/Linux so I assume it is worth to mention it.

Signed-off-by: 林博仁 &lt;<Buo.Ren.Lin@gmail.com>&gt;

This PR supersedes #1742 to match the current markup

### CODE REVIEW

**Comment by jeremejevs** [comment]
You should mention WoeUSB in [`lib/shared/messages.js`](https://github.com/resin-io/etcher/blob/master/lib/shared/messages.js) as well.

**Comment by Lin-Buo-Ren** [comment]
I have modified `lib/shared/messages.js` to add the requested changes, please review.

File: docs/USER-DOCUMENTATION.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ images, usually available from the image publishers themselves.
 Images known to require special treatment:
 
 - Microsoft Windows (use [Windows USB/DVD Download Tool][windows-usb-tool], or
-  [Rufus][rufus]).
+  [Rufus][rufus], or [WoeUSB][woeusb]).
```
Comment (jhermsmeier) [inline docs/USER-DOCUMENTATION.md]: Can you remove the ", or" before the Rufus link? It reads a bit odd otherwise

Comment (Lin-Buo-Ren) [reply docs/USER-DOCUMENTATION.md]: I suppose the comma-space shouldn't be removed?

Comment (Lin-Buo-Ren) [reply docs/USER-DOCUMENTATION.md]: I've applied the request change (with comma-space keep intact), please review.

**Comment by jhermsmeier** [comment]
Hey @Lin-Buo-Ren, sorry it took so long; it went under in between all the other things. I've resolved the conflicts & merged it now.

**Pull request merged**